stepsfunction/handler.py

```diff
@@ -20,6 +24,13 @@
 ENDMARK = b"\xff\xd9"
 ENDFIX = 2
 
+
+# setting for tesseract
```

Comment:
If you're running this locally and Tesseract is on your PATH and works fine from the CLI, you shouldn't need this. You're bundling it so it can work with Lambda, right? For testing, maybe that's not needed. Up to you.

Reply:
oops I meant to make this a single comment, not Start A Review -- I've never been sure what the difference was in GitHub :-(